### ISSUE TITLE
Stuttering / low framerate

### ISSUE THREAD

**Issue by jaxankey** [phw/peek#618]
Interesting app!

I get extremely low / stuttering frame rate when trying to record even a few seconds of gif, even on a small window. If I try byzanz, full screen is no problem at high framerate for as long as I want.

I'm on Linux Mint 19.3 Cinnamon, and get the same results with a dell ultrabook as my "beefy" gaming computer.

Is there a trick? Is it possible to use whatever byzanz uses as a backend? Peek could even just send the screen capture coordinates to byzanz via command line (as maybe a custom command option in preferences).

Cheers,
Jack

**Comment by GameKing505** [comment]
I have the same issue. Tons of frames are being missed with each recording - I don't know why.

**Comment by AlexDaniel** [comment]
Yep, it does drop frames and the animation is not smooth at all. Lowering the framerate helps a bit, but not much.

**Comment by AlexDaniel** [comment]
I also noticed that the produced GIF is very smooth in the beginning (for a few seconds), but gets choppy afterwards. It feels like there's some kind of buffer that gets filled and it is unable to keep up, or something like that.

**Comment by Mupli** [comment]
same here.

**Comment by Mupli** [comment]
is peek dead?

**Comment by AlexDaniel** [comment]
@Mupli pretty much, yeah. It's unfortunate because this stuttering is the only problem I have with it, otherwise, it's a wonderful tool. There isn't any fundamental limitation as to why it stutters, because it is able to record the first 2-3 seconds of the GIF perfectly.

Anyway, whenever I need smooth gifs now, I use [menyoki](https://github.com/orhun/menyoki). But menyoki is not able to record the mouse cursor, which makes some gifs completely pointless :(
